recipe/bld.bat

```diff
@@ -6,10 +6,8 @@ if "%NEED_SCRIPTS%" == "no" (
 )
 del %SP_DIR%\anaconda_anon_usage\plugin.py
 if not exist %PREFIX%\etc\conda\activate.d mkdir %PREFIX%\etc\conda\activate.d
-if not exist %PREFIX%\python-scripts mkdir %PREFIX%\python-scripts
 copy scripts\activate.sh %PREFIX%\etc\conda\activate.d\%PKG_NAME%_activate.sh
 copy scripts\activate.bat %PREFIX%\etc\conda\activate.d\%PKG_NAME%_activate.bat
-copy scripts\post-link.sh %PREFIX%\python-scripts\.%PKG_NAME%-post-link.sh
-copy scripts\post-link.bat %PREFIX%\python-scripts\.%PKG_NAME%-post-link.bat
-copy scripts\pre-unlink.sh %PREFIX%\python-scripts\.%PKG_NAME%-pre-unlink.sh
-copy scripts\pre-unlink.bat %PREFIX%\python-scripts\.%PKG_NAME%-pre-unlink.bat
+if not exist %PREFIX%\Scripts mkdir %PREFIX%\Scripts
+copy scripts\post-link.bat %PREFIX%\Scripts\.%PKG_NAME%-post-link.bat
+copy scripts\pre-unlink.bat %PREFIX%\Scripts\.%PKG_NAME%-pre-unlink.bat
```

Comment:
Because we're building arch packages we should be putting the scripts into `Scripts\` instead of `python-scripts`.

We include the `activate.sh` shell script for Bash-for-Windows use cases